crates/aiken-lang/src/parser/expr/if_else.rs

```diff
@@ -0,0 +1,68 @@
+use chumsky::prelude::*;
+
+use crate::{
+    ast,
+    expr::UntypedExpr,
+    parser::{error::ParseError, token::Token},
+};
+
+use super::block;
+
+pub fn parser<'a>(
+    seq_r: Recursive<'a, Token, UntypedExpr, ParseError>,
+    r: Recursive<'a, Token, UntypedExpr, ParseError>,
+) -> impl Parser<Token, UntypedExpr, Error = ParseError> + 'a {
+    just(Token::If)
+        .ignore_then(r.clone().then(block(seq_r.clone())).map_with_span(
```

Comment:
This is interesting by the by. It sounds like something as follows:

```
if foo (14) else (42)
```

would parse just fine. This isn't to be fixed in that PR as we agreed not changing the semantic, but possibly open a ticket to remember or better, add a test case that shows it parses with a bit "TODO" about making sure it eventually doesn't. I think that's a case where we do want to enforce curly braces for block-expression.